### DISCUSSION TITLE
Disable Automatic Application Updates.

### DISCUSSION THREAD

**Discussion by savidudu** [keepassxreboot/keepassxc#8419] [Q&A] [answered]
Is your Auto Update Server a public server. We would like to block the IP/FQDN for auto update Server on our Firewall to avoid Automatic Application updates ?

Can we disable the automatic updates from the keepassxc.ini file or from registry ?

We would like to disable the automatic update before installing because we have thousands of machines and after installation going on every machine and disable the auto update is bit difficult for us ?

I would like to request you to please provide all the available options for disabling the automatic updates.

Thanks

**Comment by phoerious** [comment, marked as answer]
You can set `CheckForUpdates=false` under `[GUI]` and `UpdateCheckMessageShown=true` under `[General]`. If you build KeePassXC yourself, you can also compile it with `-DWITH_XC_UPDATECHECK=OFF` to disable update checks permanently.

I would not recommend blocking the IP address, since it's a public GitHub CDN IP.

**Reply by savidudu** [reply]
Thanks for the answer, Still If you can provide the details for IP and we would like to test it on the network level.

Also, I am not seeing this option ""You can set **CheckForUpdates=false** under [GUI] and **UpdateCheckMessageShown=true** under [**General**]."" 

I mean in General I can see only Check for updates at application startup once per week which is marked as True (Tick Marked) That needs to de-selected correct ?. Could you please help me with the steps for **UpdateCheckMessageShown=true** 

Thanks,

**Reply by phoerious** [reply]
You need to set those options in the keepassxc.ini.

**Reply by savidudu** [reply]
So basically this needs to be set manually on all the machines, Correct ?

There is no way to use the main build which we will download it from the KeePassxc site and make the changes in that file ? Apart from the other option which you have highlighted ""If we are building ourselves""

Also, Could you please provide the IP/FQDN so we can try testing on the network level ?

**Reply by phoerious** [reply]
You have to provision a keepassxc.ini on all machines or use the portable ZIP file, which comes with its own keepassxc.ini.
The domain is `api.github.com`, but it seems to be round-robin DNS and you will be blocking all other GitHub API traffic.

**Reply by savidudu** [reply]
1) How to do this ""If you build KeePassXC yourself, you can also compile it with -DWITH_XC_UPDATECHECK=OFF to disable update checks permanently.""

2) From where I can get this ""**use the portable ZIP file, which comes with its own keepassxc.ini.**"" ?
Could you please point me .
Thanks a lot !

**Reply by phoerious** [reply]
See here https://github.com/keepassxreboot/keepassxc/wiki/Building-KeePassXC
and here https://keepassxc.org/download/#windows
